### PR TITLE
docs: update Chroma usage docs example

### PR DESCRIPTION
**Pull request by Alw3ys**
- Updated Chroma example to use native langchainjs query func
- Added example usage for query docs from existing collection on Chroma

### CODE REVIEW

**Review by hwchase17 — COMMENTED** [review]
i actually think we want to keep this section focused on chroma, irregardless of how its used in downstream chains. so im all for adding mroe examples of chroma.from_documents, etc, but i dont think we should use in chains (theres other sections of the docs for that)

**Comment by Alw3ys** [comment]
> i actually think we want to keep this section focused on chroma, irregardless of how its used in downstream chains. so im all for adding moe examples of chroma.from_documents, etc, but i dont think we should use in chains (theres other sections of the docs for that)

For sure, updated it! @hwchase17

File: docs/docs/modules/indexes/vector_stores/chroma.md [hunk 1]
```diff
@@ -13,32 +13,31 @@ Use [chroma](https://github.com/chroma-core/chroma) with langchainjs.
...
 ```
 
-## Usage
+## Index and query docs
 
 ```typescript
 import { Chroma } from "langchain/vectorstores";
 import { OpenAIEmbeddings } from "langchain/embeddings";
 
-// text sample from Godel, Escher, Bach
```
Comment (nfcampos) [inline docs/docs/modules/indexes/vector_stores/chroma.md:22]: Can we just keep both? These are two valid ways of using Chroma VectorStore

**Review by hwchase17 — APPROVED** [review]
lgtm! thanks!

**Pull request merged**